### PR TITLE
Create test users in demo site

### PR DESCRIPTION
**Pull request by alvinsng**
This updates our demo site to support creation of test users

### CODE REVIEW

File: pages/api/create-order.ts [hunk 1]
```diff
@@ -24,7 +25,13 @@ export default async function handler(
       total,
       items,
       customerId,
+      vendor: {
```
Comment (alvinsng) [inline pages/api/create-order.ts:27]: @gpsamson added fields used by PQ

File: pages/api/create-order.ts [hunk 2]
```diff
@@ -24,7 +24,13 @@ export default async function handler(
...
+        foo: "bar",
```
Comment (gpsamson) [inline pages/api/create-order.ts:28]: More closely mimicks what is passed in
```suggestion
        id: "foo",
```

**Review by gpsamson — APPROVED** [review]
See single comment!

File: pages/api/create-order.ts (hunk 2 above)
Comment (alvinsng) [reply pages/api/create-order.ts:28]: I think foo is a better indicate it can be anything

**Pull request merged**